server/controllers/search/lib/entities_query_builder.js

```diff
@@ -48,68 +52,63 @@ module.exports = params => {
 }
 
 const matchEntities = (search, userLang, exact, safe) => {
-  // In case a claim alone is searched
-  if (search == null) return
-
-  const fields = entitiesFields(userLang, exact)
-
-  // From time to time, cross_fields generates 'function score query returned an invalid score' errors
-  // See https://github.com/elastic/elasticsearch/issues/44700
-  // So, until there is a fix for that, requests that generate those errors will be retried in "safe" mode,
-  // that is, with best_fields instead of cross_fields
-  const matchType = safe ? 'best_fields' : 'cross_fields'
-
-  const queries = [
-    {
-      // Use query_string to give exact matches a boost.
-      // See query strings doc : https://www.elastic.co/guide/en/elasticsearch/reference/7.10/query-dsl-query-string-query.html
+  if (exact) {
+    return {
       multi_match: {
         query: search,
         operator: 'and',
-        fields,
-        analyzer: 'standard_truncated',
-        type: matchType,
-        boost: 3
+        fields: exactMatchEntitiesFields(userLang),
+        analyzer: 'standard_full',
+        type: 'best_fields',
       }
     }
-  ]
-
-  if (!exact) {
-    queries.push({
+  } else {
+    return {
```

Comment:
advantage of having the same multi_match for both exact and !exact was that exact must have a superior score to !exact (which is nice in every case (?)) 
drawback is exact performance, since 2 queries a fired

i would rather be for a little changes at a time and keep a same `multi_match` query for both exact and !exact matches

Reply:
done in 1372e549